Manifest.toml

```diff
@@ -0,0 +1,249 @@
+# This file is machine-generated - editing it directly is not advised
+
+[[Base64]]
```

Comment:
Add the `Manifest.toml` file in `.gitignore`

Reply:
Good catch, I was always adding this file to repos. Thanks!

Reply:
Done!